### DISCUSSION TITLE
Prevent non-admin users from being able to change settings

### DISCUSSION THREAD

**Discussion by wraithy** [diced/zipline#399] [Ideas]
It would be an excellent addition if you were able to prevent users from being able to change certain settings so that you can keep your URLs unified. The main settings that I am talking about are the embed title, embed color, embed site name and domains. It would also be nice to be able to lock the file name format to a specific format for all users instead of them being able to change it. 

I think that all of the things mentioned above should be added to some sort of toggleable permission-based system that can be modified by administrators to allow them to choose how they want their instance of zipline to work. More options such as disabling certain stats for non-admin users would be nice too for privacy reasons. It would also be nice if you could see and manage what people upload to deal with DMCA related issues etc.

**Comment by superintendent2521** [comment]
administrators can view people's uploads now via a fork made.
